packages/devtools-node/src/fs.ts

```diff
@@ -34,7 +34,7 @@ export async function readEncodedComposite(
   ceramic: CeramicClient | string,
   path: PathInput
 ): Promise<Composite> {
-  const client = ceramic instanceof CeramicClient ? ceramic : new CeramicClient(ceramic)
+  const client = typeof ceramic === 'string' ? new CeramicClient(ceramic) : ceramic
```

Comment:
Can you revert to using `instanceof` please?

Reply:
Yes! Sorry, I somehow missed that. Thanks for checking, Paul :).